core/libs/talawa-request-context.js

```diff
@@ -0,0 +1,67 @@
+const cls = require('cls-hooked');
+const clsBluebird = require('cls-bluebird');
+const i18n = require('i18n');
+
+const requestContextNamespace = cls.createNamespace('talawa-request-context');
+clsBluebird(requestContextNamespace);
+
+const setRequestContextValue = (key, value) => {
+  return requestContextNamespace.set(key, value);
+};
+
+const getRequestContextValue = (key) => {
+  return requestContextNamespace.get(key);
+};
+
+const setRequestContext = (obj) => {
+  setRequestContextValue('translate', obj.__);
+  setRequestContextValue('translatePlural', obj.__n);
+};
+
+const middleware = () => {
+  return (req, res, next) => {
+    requestContextNamespace.bindEmitter(req);
+    requestContextNamespace.bindEmitter(res);
+
+    requestContextNamespace.run(() => {
+      setRequestContext(req);
+      next();
+    });
+  };
+};
+
+const init = async (options = {}) => {
+  const obj = {};
+  i18n.init(obj);
+  obj.setLocale(options.lang);
+  return requestContextNamespace.runAndReturn(async () => {
+    setRequestContext({
+      __: obj.__,
+      __n: obj.__n,
+    });
+    return options.requestHandler();
+  });
+};
+
+const translate = (...args) => {
```

Comment:
There is no as such json file for error message translation. Can you add a demo of how to throw error in some other language like French, Hindi or Spanish